packages/slice-machine/src/features/onboarding/SliceMachineOnboardingGuide/OnboardingProgressStepper.tsx

```diff
@@ -35,9 +36,10 @@ export function OnboardingProgressStepper(
     setActiveStep(step);
     setDialogOpen(true);
     void telemetry.track({
-      event: "onboarding:step-opened",
+      event: "shared-onboarding-guide:step-open",
```

Comment:
⚠️ That's wrong no? If it's the new guide it's good but if the old guide is displayed here it will sent the wrong event and mislead product analysis, right?

Reply:
That was discussed with Come, I think we need  the same events so we can compare experiment variants

Reply:
I don't like the "I think" because we need to be sure 😛

Reply:
Changed to use old/current events ✅